### ISSUE TITLE
clean up code in use_DRAC()

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#435]
There are some patterns in `use_DRAC()` that could be improved:

- https://github.com/R-Lum/Luminescence/blob/670fd1c43a9b6ead80ace897667d58085bf0d956/R/use_DRAC.R#L135
This is meant to pick a few letters (2 or 3 effectively) at random to generate some identifier: I could simply be `sample(c(LETTERS, letters), runif(1, 2, 4))`, as I don't think it matters that letters should have the same capitalization

- https://github.com/R-Lum/Luminescence/blob/670fd1c43a9b6ead80ace897667d58085bf0d956/R/use_DRAC.R#L223
This could just be `sample(c("-", " "), 1)`, and we could add "_" too as a possibility for a bit more variation

- https://github.com/R-Lum/Luminescence/blob/670fd1c43a9b6ead80ace897667d58085bf0d956/R/use_DRAC.R#L224-L225
This gets a sequence of consecutive numbers starting from a random one and stores it as 2-digit string with the first being a 0 if the number only has 1 digit. Here I don't think it is important that in `sample()` the lowest numbers should have higher probability; also the printing could be achieved with `sprintf()`. So we could do this instead: `sprintf("%02d", seq(sample(1:50, 1), by = 1, length.out = nrow(DRAC_submission.df))`

- https://github.com/R-Lum/Luminescence/blob/670fd1c43a9b6ead80ace897667d58085bf0d956/R/use_DRAC.R#L224-L225
One single `paste0()` is enough in this case

**Comment by mcol** [comment]
The pattern is immediately retrievable:
```
Real IDs:[1] "r01"
All IDs:[1] "r01" "e02" "r03" "e04"

Real IDs:[1] "f28"
All IDs:[1] "f28" "t29" "f30" "t31"

Real IDs:[1] "m08" "k09" "m10"
All IDs: [1] "m08" "k09" "m10" "k11" "m12" "k13" "m14" "k15" "m16" "k17" "m18" "k19"

Real IDs:[1] "c02" "t03" "q04"
All IDs: [1] "c02" "t03" "q04" "c05" "t06" "q07" "c08" "t09" "q10" "c11" "t12" "q13"
```

**Comment by mcol** [comment]
I can now make it like this, which is not so easily crackable:
```
Real IDs:[1] "m988"
Shuffled:[1] "Q_491" "Q818"  "m-523" "m988"

Real IDs:[1] "u_414"
Shuffled:[1] "A-116" "u061"  "u_414" "A_768"

Real IDs:[1] "b680"  "C-170" "b_192"
Shuffled: [1] "b_338" "b-572" "C_225" "b-501" "C-170" "C000"  "C983"  "b680"  "C-245" "b_192" "C_743" "b868"

Real IDs:[1] "u045"  "Y_657" "B-065"
Shuffled: [1] "u897"  "Y_268" "u045"  "B-879" "B-417" "B-065" "u678"  "Y_657" "u447" "Y_415" "Y_602" "B-859"
```

**Comment by mcol** [comment]
Fixed by #438.
